rever/activities/conda_forge.xsh

```diff
@@ -2,6 +2,7 @@
 import os
 import re
 import sys
+import requests
```

Comment:
You should not need to use requests directly. Github3.py almost certainly has a way to fork a repo

Reply:
https://github3py.readthedocs.io/en/master/repos.html#github3.repos.repo.Repository.create_fork

Reply:
I'm using GitHub3 but I don't know if that allows me to check if the fork already exists.

Reply:
Yes, you can either list the public repos (https://github3py.readthedocs.io/en/master/github.html#github3.github.GitHub.iter_user_repos) or all repos (https://github3py.readthedocs.io/en/master/github.html#github3.github.GitHub.iter_repos).

Or you can perhaps try to get the feedstock repo itself and see if it fails (https://github3py.readthedocs.io/en/master/github.html#github3.github.GitHub.repository)

Reply:
I guess I can also try except forking the repo, if I can't fork it it must exist?

Reply:
Yes, but that would be bad form since the whether or not the repo exists is knowable and exception s can happen for other reasons too :)

Reply:
I think I'm going with option 3) (the get repo, if `None` fork option)